src/Enum/ModelValidationStrategy.php

```diff
@@ -0,0 +1,500 @@
+use ToshY\BunnyNet\Model\Api\Shield\Waf\ReviewTriggeredRuleAiRecommendation;
+use ToshY\BunnyNet\Model\Api\Shield\Waf\ReviewTriggeredRules;
+use ToshY\BunnyNet\Model\Api\Shield\Waf\UpdateCustomWafRule;
+use ToshY\BunnyNet\Model\Api\Shield\Waf\UpdateCustomWafRuleByPatch;
+use ToshY\BunnyNet\Model\Api\Shield\Zone\CreateShieldZone;
+use ToshY\BunnyNet\Model\Api\Shield\Zone\GetShieldZone;
+use ToshY\BunnyNet\Model\Api\Shield\Zone\GetShieldZoneByPullZoneId;
+use ToshY\BunnyNet\Model\Api\Shield\Zone\ListShieldZones;
+use ToshY\BunnyNet\Model\Api\Shield\Zone\UpdateShieldZone;
+use ToshY\BunnyNet\Model\Api\Stream\ManageCollections\CreateCollection;
+use ToshY\BunnyNet\Model\Api\Stream\ManageCollections\DeleteCollection;
+use ToshY\BunnyNet\Model\Api\Stream\ManageCollections\GetCollection;
+use ToshY\BunnyNet\Model\Api\Stream\ManageCollections\ListCollections;
+use ToshY\BunnyNet\Model\Api\Stream\ManageCollections\UpdateCollection;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\AddCaption;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\AddOutputCodecToVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\CreateVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\DeleteCaption;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\CleanupUnconfiguredResolutions;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\DeleteVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\FetchVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\GetVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\GetVideoHeatmap;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\GetVideoPlayData;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\VideoResolutionsInfo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\ListVideos;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\GetVideoStatistics;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\ReEncodeVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\RepackageVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\SetThumbnail;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\TranscribeVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\UpdateVideo;
+use ToshY\BunnyNet\Model\Api\Stream\ManageVideos\UploadVideo;
+use ToshY\BunnyNet\Model\Api\Stream\OEmbed\GetOEmbed;
+use ToshY\BunnyNet\Validation\Strategy\Body\LaxBodyValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\Body\NoBodyValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\Body\StrictBodyValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\Query\LaxQueryValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\Query\NoQueryValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\Query\StrictQueryValidationStrategy;
+use ToshY\BunnyNet\Validation\Strategy\ValidationModelStrategy;
+
+enum ModelValidationStrategy
+{
+    case STRICT;
+
+    case STRICT_QUERY;
+
+    case STRICT_BODY;
+
+    case LAX;
+
+    case NONE;
+
+    /** @var array<class-string,ModelValidationStrategy> */
+    private const BASE = [
```

Comment:
Maybe, not sure, if can update the generator in a way to update the `ModelValidationStrategy` as well. Think it's possible, but need to check if it will recreate it correctly regarding the strategy.